code/__DEFINES/clothing.dm

```diff
@@ -49,6 +37,49 @@
 
 #define slots_amt			20 // Keep this up to date!
 
+//I hate that this has to exist

```

Comment:
Why does it exist then? I can't quite grasp the difference between slots and SLOTS